composer.json

```diff
@@ -14,8 +14,8 @@
   },
   "require": {
     "php": "^7.1",
-    "symfony/console": "^4.0",
-    "symfony/finder": "^4.0",
+    "symfony/console": "^4.0 || ^5.0",
```

Comment:
How come your using the `||` operator here? Cant we just update it to 5.0 and then bump up the major release version?

Reply:
this PR (and this comment) is a duplicate of #102

Reply:
I just learned it. #102 has title "fixes #98" which does not speak much